pytomo3d/window/filter_windows.py

```diff
@@ -213,21 +234,27 @@ def filter_windows_on_measurements(windows, measurements, measure_config):
     if len(windows) == 0:
         return {}, {}
 
-    means, stds = get_measurements_std(measurements)
+    dt_means, dt_stds, dlna_means, dlna_stds = \
+        get_measurements_std(measurements)
 
     # get the measurement bounds
     final_bounds = {}
-    for comp in means:
+    for comp in dt_means:
         print("-" * 20 + "\nComponent: %s" % comp)
         user_bound = get_user_bound(comp_config[comp])
-        std_bound = get_std_bound(means[comp], stds[comp],
-                                  comp_config[comp]["std_ratio"])
-
-        bound = [max(std_bound[0], user_bound[0]),
-                 min(std_bound[1], user_bound[1])]
+        dt_std_bound = get_std_bound(dt_means[comp], dt_stds[comp],
+                                     comp_config[comp]["std_ratio"])
+        dlna_std_bound = get_std_bound(dlna_means[comp], dlna_stds[comp],
+                                       comp_config[comp]["std_ratio"])
```

Comment:
I thought about it but I think it would be better to set the criteria of standard deviation ratio to the same rather than different.  Hardly see the reason to set one stricter than the other in terms of excluding the outliers